src/gfn/gflownet/base.py

```diff
@@ -167,7 +210,12 @@ def get_trajectories_scores(
         total_log_pf_trajectories = log_pf_trajectories.sum(dim=0)
         total_log_pb_trajectories = log_pb_trajectories.sum(dim=0)
 
-        log_rewards = trajectories.log_rewards.clamp_min(self.log_reward_clip_min)  # type: ignore
+        log_rewards = trajectories.log_rewards
+        if math.isfinite(self.log_reward_clip_min) and not isinstance(
+            log_rewards, type(None)
+        ):
+            log_rewards = log_rewards.clamp_min(self.log_reward_clip_min)
```

Comment:
Where is `self.log_reward_clip_min` defined ?

Reply:
nice catch thanks --

I guess it would be cleaner to move this to the base class. I'll leave a TODO for now.